__std__/plutus-apps/packages/pre-commit-check.nix

```diff
@@ -20,11 +20,6 @@ inputs.pre-commit-hooks-nix.lib.run {
     cabal-fmt.enable = true;
     shellcheck.enable = true;
 
-    editorconfig-checker = {
```

Comment:
why? undesirable?

Reply:
plutus-apps doesn't currently have `editorconfig-checker`.
I run it and I got literally over 9000 failures.
If the team wants to use it'll have to done in a separate PR